themes/frontierline/style.css

```diff
@@ -18,729 +18,3029 @@ Domain Path: /languages/
 *
 * Don't edit style.css directly!
 **********/
-@font-face { font-family: 'Zilla Slab'; font-weight: normal; font-style: normal; src: url("fonts/ZillaSlab-Regular.woff2") format("woff2"), url("fonts/ZillaSlab-Regular.woff") format("woff"); }
```

Comment:
The diff is huge. Try `sass --style compact --sourcemap=auto ./sass/style.scss style.css` in the theme root. @craigcook can you add the correct sass command in the readme, please? I am not sure if this one is the best to use either.

Reply:
I've used compact style. I'm not sure how much difference it really makes in the end (I haven't compared the file size before and after) but it definitely makes the diff seem bigger than it is just from all the line changes.

I can update the readme. It's rare for this theme to get this much activity that we'd even need to worry about it, it's usually just me :)

Reply:
Updated with the suggested command but the diff is just as big and now there's a bunch of extra line breaks in it too.

Reply:
Oh wait, I did it on the other PR >_<